batch/batch/worker/worker.py

```diff
@@ -1047,13 +1053,7 @@ async def _run_container(self) -> bool:
 
                     assert self.netns
 
-                    async with ResourceUsageMonitor(
-                        self.name,
-                        self.container_overlay_path,
-                        self.io_mount_path,
-                        self.netns.veth_host,
-                        self.resource_usage_path,
-                    ):
+                    async with monitor:
```

Comment:
I don't think we should stop monitoring a JVM's resources. In fact, I think we should eventually have a per-worker page that includes information like JVM resource usage, JVM logs, and other worker metadata.

If we leave resource monitoring here, as it already is, I think it keeps the code simpler. Let's do that.